cross/toolchain.cmake

```diff
@@ -76,8 +76,6 @@ if (CLR_CMAKE_COMPILER STREQUAL "Clang")
   add_compile_param(CROSS_LINK_FLAGS "--target=${TOOLCHAIN}")
 endif()
 
-add_compile_param(CROSS_LINK_FLAGS "-fuse-ld=gold")
```

Comment:
could this impact other cross-compilations?

Reply:
It would change the linker they are built with to regular ld (unless we have PGO data for them, which turns on the ld.gold). But that should be fine.